ui/css/main.css

```diff
@@ -439,19 +360,5 @@ i.fa-plus{
     font-size: 3em;
     text-align: center;
 }
-.car_image img{
-    width: 200px;
-    height: 200px;
-    border-radius: 50%;
-    
-}
-.car_image{
-    width: 100%
-}
 
-/*/* Force table to not be like tables anymore */ 
-/*@media only screen and (max-width: 500px){*/
-  /*table, thead, tbody, th, td, tr {*/
-      /*display: block; */
```

Comment:
I believe this commented out block of code is of no use any longer. Consider removing it.